File: DailyFlash Solutions/DailyFlash Solutions/74-DailyFlash_Solutions/15_April_Solution_One/Java/prog5.java
import java.util.*;

class Demo {
	public static void main(String[] args) {
		int m;
		System.out.println("m: ");
		Scanner sc = new Scanner(System.in);
		m = sc.nextInt();
		int arr1[][] = new int[m][m];
		int arr2[][] = new int[m][m];
		int arr[][] = new int[m][m];
		System.out.println("Enter arr1: ");
		for(int i = 0; i < m; i++) {
			for(int j = 0; j < m; j++) {
				arr1[i][j] = sc.nextInt();;
			}
		}
		System.out.println("Enter arr2: ");
		for(int i = 0; i < m; i++) {
			for(int j = 0; j < m; j++) {
				arr2[i][j] = sc.nextInt();
				arr[i][j] = arr1[i][j] + arr2[i][j];
			}
		}	
		
		System.out.println("arr: ");
		for(int i = 0; i < m; i++) {
			for(int j = 0; j < m; j++) {
				System.out.println(arr[i][j]);
			}
			System.out.print("\n");
		}	

		
		
	}
}

File: DailyFlash Solutions/DailyFlash Solutions/74-DailyFlash_Solutions/15_April_Solution_One/C++/prog5.cpp
#include<iostream>

int main() {
	int m;
	std::cout << "m: ";
	std::cin >> m;
	int arr1[m][m];
	int arr2[m][m];
	int arr[m][m];
	std::cout << "Enter arr1: ";
	for(int i = 0; i < m; i++) {
		for(int j = 0; j < m; j++) {
			std::cin >> arr1[i][j];
		}
	}
	std::cout << "Enter arr2: ";
	for(int i = 0; i < m; i++) {
		for(int j = 0; j < m; j++) {
			std::cin >> arr2[i][j];
			arr[i][j] = arr1[i][j] + arr2[i][j];
		}
	}	
	
	std::cout << "arr: ";
	for(int i = 0; i < m; i++) {
		for(int j = 0; j < m; j++) {
			std::cout << arr[i][j];
		}
		std::cout << "\n";
	}	

	
	
}
